src/instruments/instrumentstypes.h

```diff
@@ -171,6 +180,37 @@ struct PartInstrument {
 
 using PartInstrumentList = QList<PartInstrument>;
 
+struct ScoreOrderGroup {
+    int index;
+    QString family;
+    QString section { QString() };
+    QString unsorted { QString() };
+    bool bracket { false };
```

Comment:
Better to use = here

bracket = false;
showSystemMarkings = false;
...

Reply:
Done